ern-container-gen-android/src/hull/gradlew

```diff
@@ -80,10 +80,10 @@ do
     esac
 done
 
-APP_HOME=$( cd "${APP_HOME:-./}" && pwd -P ) || exit
-
-APP_NAME="Gradle"
+# This is normally unused
+# shellcheck disable=SC2034
 APP_BASE_NAME=${0##*/}
+APP_HOME=$( cd "${APP_HOME:-./}" && pwd -P ) || exit
```

Comment:
Do these changes possibly come from one of the later versions (8?) of Gradle? I don't see them with 7.3 (or 7.3.3) - How did you update the files?

Reply:
updating back to 7.3.3.